server/package.json

```
@@ -11,14 +11,16 @@
   "contributors": [
     "Benjamin Naesen <benjamin.naesen@studiohyperdrive.be>",
     "Bart Naessens <bart.naessens@studiohyperdrive.be>",
-    "Ian Emsens <ian.emsens@studiohypderive.be>"
+    "Ian Emsens <ian.emsens@studiohypderive.be>",
+    "Ward Vercruyssen <ward.vercruyssen@studiohypderive.be>"
   ],
   "homepage": "https://github.com/viaacode/avo2-proxy",
   "scripts": {
     "start": "npm run serve",
     "start-with-env": "dotenv -e .env/.env npm run serve",
     "start-with-env-dev": "dotenv -e .env/.env.dev npm run serve",
     "start-with-env-qas": "dotenv -e .env/.env.qas npm run serve",
+    "dev": "npm run start-with-env-qas",
```

Avoid confusion with `start-with-env-dev`

```suggestion
    "qas": "npm run start-with-env-qas",
```